app/src/main/java/org/oppia/android/app/topic/TopicActivityPresenter.kt

```diff
@@ -39,6 +39,17 @@ class TopicActivityPresenter @Inject constructor(
         topicFragment, TOPIC_FRAGMENT_TAG
       ).commitNow()
     }
+
+    if (getSpotlightFragment() == null) {
```

Comment:
As suggested by Ben in one thread above, create `newInstance` companion function inside the fragment to create this fragment anywhere. Have a look at `TopicInfoFragment` for clarity.

Reply:
done